File: output/GPT3.5-Turbo/xCodeEval_4000/C++/C/cc71589cdb8b0c04028e87bc7f1cb2e7.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#define MULTITEST

#define N 1000 + 7

typedef long long li;

int n;
char s[N];

li lim[] = {0, 9, 99, 99, 3099, 13099, 113099, 1113099, 11113099, 111113099};
char pr1[][4] = {"", "198", "19", "2", "", "", "", "", "", ""};
char pr2[][4] = {"", "199", "20", "3", "1", "1", "1", "1", "1", "1"};

char* solve() {
    li x = atoll(s);
    if (x >= lim[n])
        return strcat(pr1[n], s);
    else
        return strcat(pr2[n], s);
}

int read() {
    if (scanf("%s", s) != 1)
        return 0;
    strcpy(s, s + 4);
    n = strlen(s);
    return 1;
}

int main() {
#ifdef _DEBUG
    freopen("input.txt", "r", stdin);

    int tt = clock();
#endif

    printf("%.15lf\n", 0.0);
    printf("%.15lf\n", 0.0);

#ifdef MULTITEST
    int tc;
    scanf("%d", &tc);
    while(tc--){
        read();
#else
    while(read()) {    
#endif
        puts(solve());
        
#ifdef _DEBUG
    printf("TIME = %d\n", clock() - tt);
    tt = clock();
#endif

    }
}

File: output/GPT3.5-Turbo/xCodeEval_4000/C++/Python/cc71589cdb8b0c04028e87bc7f1cb2e7.py
import sys
import math
import random

def solve():
    x = int(s)
    if x >= lim[n]:
        return pr1[n] + s
    else:
        return pr2[n] + s

def read():
    s = input().strip()
    s = s[4:]
    n = len(s)
    return s, n

def main():
    sys.stdin = open('input.txt', 'r')
    sys.stdout = open('output.txt', 'w')

    tc = int(input())
    for _ in range(tc):
        s, n = read()
        print(solve())

if __name__ == "__main__":
    main()
